pkg/operator/vspherecontroller/checks/check_error.go

```diff
@@ -124,6 +147,11 @@ func CheckClusterStatus(result ClusterCheckResult, apiDependencies KubeAPIInterf
 		}
 
 		return ClusterCheckBlockUpgrade, result
+
+	case CheckActionBlockInstall:
+		return ClusterCheckBlockInstall, result
```

Comment:
`ClusterCheckBlockInstall` should may be renamed to `ClusterCheckBlockDriverInstall`. I know it is bit mountful but perhaps more accurate.

Reply:
done